tools/twix/src/panels/map/layers/ball_filter.rs

```diff
@@ -28,27 +28,23 @@ impl Layer for BallFilter {
         let ball_hypotheses = nao.subscribe_output(CyclerOutput {
             cycler: Cycler::Control,
             output: Output::Additional {
-                path: "ball_filter_hypotheses".to_string(),
+                path: "best_ball_state".to_string(),
             },
         });
         Self {
             robot_to_field,
-            ball_hypotheses,
+            ball_state: ball_hypotheses,
         }
     }
 
     fn paint(&self, painter: &TwixPainter, _field_dimensions: &FieldDimensions) -> Result<()> {
         let robot_to_field: Option<Isometry2<f32>> = self.robot_to_field.parse_latest()?;
-        let ball_hypotheses: Vec<Hypothesis> = self.ball_hypotheses.parse_latest()?;
-
-        for hypothesis in ball_hypotheses.iter() {
-            let position =
-                robot_to_field.unwrap_or_default() * Point2::from(hypothesis.state.mean.xy());
-            let covariance = hypothesis
-                .state
-                .covariance
-                .fixed_view::<2, 2>(0, 0)
-                .into_owned();
+        let best_ball_state: Option<MultivariateNormalDistribution<4>> =
```

Comment:
I think we should add back all detected hypotheses back to twix. However, considering current priorities, I suggest creating an issue to add all hypotheses back to twix at some later point in time.